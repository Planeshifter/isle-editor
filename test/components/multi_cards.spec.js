// MODULES //

import React from 'react';
import renderer from 'react-test-renderer';
import MultiCards from '@isle-project/components/multi-cards/main.js';


// TESTS //

describe( '<MultiCards />', function test() {
	it( 'renders a multiple cards component', () => {
		const cards = renderer.create(
			<MultiCards
				values={[
					{ front: 'number 1', back: 'back side of #1' },
					{ front: 'number 2', back: 'back side of #2' },
					{ front: 'number 3', back: 'back side of #3' },
					{ front: 'number 4', back: 'back side of #4' }
				]}
			/>
		).toJSON();
		expect( cards ).toMatchSnapshot();
	});
});
