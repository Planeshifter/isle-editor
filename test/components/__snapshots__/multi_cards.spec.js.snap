// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultiCards /> renders a multiple cards component 1`] = `
<div
  style={
    Object {
      "animationDuration": undefined,
      "animationName": undefined,
      "overflow": "auto",
    }
  }
/>
`;
